__tests__/Unit/Components/Tasks/TaskDetails.test.tsx

```diff
@@ -191,229 +190,216 @@ describe('TaskDetails Page', () => {
+            }
         });
     });
 
-    it('Renders Task Ends-on Date', async () => {
+    it('Renders Task Assignee', async () => {
         const { getByText } = renderWithRouter(
             <Provider store={store()}>
-                <Details
-                    detailType={'EndsOn'}
-                    value={'4/19/2021, 12:00:10 AM'}
-                />
+                <TaskDetails taskID={details.taskID} />
             </Provider>
         );
         await waitFor(() => {
-            expect(getByText('4/19/2021, 12:00:10 AM')).toBeInTheDocument();
+            expect(getByText('ankur')).toBeInTheDocument();
         });
     });
-
-    it('Renders Extension Request icon', async () => {
-        const { getByText } = renderWithRouter(
-            <Provider store={store()}>
-                <Details
-                    detailType={'Ends On'}
-                    value={'4/19/2021, 12:00:10 AM'}
-                    url={details.extension_request_url}
-                />
-            </Provider>
-        );
-
-        await waitFor(() => {
-            const element = screen.queryByTestId('extension-request-icon');
-            expect(element).toHaveAttribute(
-                'href',
-                'https://dashboard.realdevsquad.com/extension-requests?order=asc&q=taskId%3AzlwjJzKbGpqCoCTZMZQy'
-            );
-            expect(getByText('4/19/2021, 12:00:10 AM')).toBeInTheDocument();
-        });
+});
+it('Renders Task Ends-on Date', async () => {
+    const { queryAllByText } = renderWithRouter(
+        <Provider store={store()}>
+            <Details detailType={'EndsOn'} value={'4/19/2021, 12:00:10 AM'} />
+        </Provider>
+    );
+    await waitFor(() => {
+        const dateElements = queryAllByText('4/19/2021, 12:00:10 AM');
+        expect(dateElements.length).toBeGreaterThan(0);
     });
+});
 
-    it('Doesnot Renders Extension Request icon when url not available', async () => {
-        const { getByText } = renderWithRouter(
-            <Provider store={store()}>
-                <Details
-                    detailType={'Ends On'}
-                    value={'4/19/2021, 12:00:10 AM'}
-                    url={null}
-                />
-            </Provider>,
-            {
-                query: { dev: 'true' },
-            }
-        );
-        const element = screen.queryByTestId('extension-request-icon');
-        expect(element).toBeNull();
-        await waitFor(() => {
-            expect(getByText('4/19/2021, 12:00:10 AM')).toBeInTheDocument();
-        });
+it('Does not render Extension Request icon when URL not available', async () => {
+    const { queryByTestId } = render(
+        <Provider store={store()}>
+            <Details
+                detailType={'Ends On'}
+                value={'4/19/2021, 12:00:10 AM'}
+                url={null}
+            />
+        </Provider>
+    );
+    await waitFor(() => {
+        expect(queryByTestId('extension-request-icon')).toBeNull();
```

Comment:
Why this part is being removed? any reason
```
await waitFor(() => {
            expect(getByText('4/19/2021, 12:00:10 AM')).toBeInTheDocument();
```

Reply:
I removed the assertion for `4/19/2021, 12:00:10 AM` because this test specifically focuses on verifying that the `extension-request-icon` is not rendered when the URL is not available. Since the presence of `4/19/2021, 12:00:10 AM` is not directly related to this aspect of the component's behaviour, I felt it was unnecessary to include it in this particular test case. However, I understand your concern and can add it back if you believe it's important for comprehensive testing. Would you like me to include it?

Reply:
If there's another test we can write to handle this, or may be change the title of the tests and include in this one too.

Reply:
What do you think would be the appropriate title for this test?

Reply:
Let's tackle this in the next PR